package.json

```diff
@@ -60,23 +60,22 @@
     }
   },
   "dependencies": {
-    "@sentry/browser": "8.42.0",
-    "@sentry/core": "8.42.0",
-    "@sentry/types": "8.42.0",
-    "@sentry/utils": "8.42.0"
+    "@sentry/browser": "9.0.0",
+    "@sentry/core": "9.0.0",
+    "@sentry/types": "9.0.0"
   },
   "devDependencies": {
     "@capacitor/android": "^3.0.1 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0",
     "@capacitor/core": "^3.0.1 || ^4.0.0 || ^5.0.0 || ^6.0.0  || ^7.0.0",
     "@capacitor/ios": "^3.0.1 || ^4.0.0 || ^5.0.0 || ^6.0.0  || ^7.0.0",
     "@ionic/prettier-config": "^1.0.0",
-    "@sentry-internal/eslint-config-sdk": "8.42.0",
-    "@sentry-internal/eslint-plugin-sdk": "8.42.0",
-    "@sentry-internal/typescript": "8.42.0",
+    "@sentry-internal/eslint-config-sdk": "9.0.0",
+    "@sentry-internal/eslint-plugin-sdk": "9.0.0",
+    "@sentry-internal/typescript": "9.0.0",
     "@sentry/wizard": "3.34.2",
     "@types/jest": "^29.5.3",
     "concurrently": "^8.2.2",
-    "eslint": "^7.13.0",
+    "eslint": "^7.6.0",
```

Comment:
Did you mean to downgrade eslint here?

Reply:
Nice catch! fixed on https://github.com/getsentry/sentry-capacitor/pull/840/commits/da2eb29901775772251ee0abed1e8476ba429ad4